### ISSUE TITLE
User model not created

### ISSUE THREAD

**Issue by Marlysson** [yatingupta10/ourpoemz#7]
Same deleting blog.db file and run again the project the table don't generate.

Should be declarated, isn't?

**Comment by jugba** [comment]
Yes the User model does not exist at the moment, I got the app running by removing it.

**Comment by yatingupta10** [comment]
yes that is also to be done!

**Comment by Marlysson** [comment]
Yes , I too.. removed it and running.

Are this project participating in Hacktoberfest? :D
